debug-commands.md

```
@@ -0,0 +1,105 @@
+```
+[CONTACT SAVE] Attaching listener for saved contact room: abc123
+[LISTENER] Attempting to attach listener for room: abc123
+[LISTENER] Attaching new listener for room: abc123 (total: 1)
+[CONTACT CALL] Ensuring listener is active for room: abc123
+[LISTENER] Duplicate listener prevented for room: abc123
+incoming call from user123 for room abc123
+```
+
+### Bug Scenario:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)